ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-ec.yaml

```diff
@@ -982,7 +982,7 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-filestore-csi
-- as: gcp-ipi-minimal-permission-f14
+- as: gcp-ipi-minimal-permission-custom-type-f14
```

Comment:
After https://github.com/openshift/release/pull/42920 get merged, I think this job does not need any more. BTW, I guess you need rebase your branch.